.azure/gpu-tests-lite.yml

```
@@ -62,18 +62,19 @@ jobs:
       displayName: 'set visible devices'
 
     - bash: |
-        set -e
-        PYTORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
-        python ./requirements/pytorch/adjust-versions.py requirements/lite/base.txt ${PYTORCH_VERSION}
-
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
         pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
```

I think these two lines can be completely removed because they're duplicates of the following step
```suggestion
        CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
        pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
```